src/main/java/uk/gov/ons/census/fwmtadapter/model/dto/ReceiptDTO.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.ons.census.fwmtadapter.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.OffsetDateTime;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class ReceiptDTO {
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  @JsonProperty("caseId")
+  private String caseId;
+
+  private String questionnaireId;
+
+  private boolean unreceipt;
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  @JsonProperty("dateTime")
```

Comment:
If this is the same dateTime that comes from the `event` in responseManagement message then this probably isn't needed?

Reply:
This is just copied from someone else's caseprocessor work, this is now the shape sent from pubsub

Reply:
Think this might need to be removed if it's not matching the [pubsub message](https://github.com/ONSdigital/census-rm-pubsub/blob/master/app/subscriber.py#L60-L75), just to be consistent? Appreciate it will be excluded if it's not there though.